collector/collector.go

```diff
@@ -355,13 +364,27 @@ type internalMetrics struct {
 	snmpRetries           prometheus.Counter
 }
 
+type NamedModule struct {
+	*config.Module
+	name string
+}
```

Comment:
Is there are point to keep Module and NamedModule separate, rather than add a "name" attribute to Module?

Reply:
I thought a light wrapper would be better than playing around with the Unmarshal process.

Reply:
OK, that makes sense.